packages/common/test/utils.spec.ts

```diff
@@ -108,4 +108,83 @@ describe('[Utils/Parse]', () => {
       'should parse correct address',
     )
   })
+
+  it('should assign correct blob schedule', async () => {
+    // clone json out to not have side effects
+    const customData = JSON.parse(JSON.stringify(postMergeHardforkData))
+    Object.assign(customData.config, {
+      "chainId": 3151908,
+      "homesteadBlock": 0,
+      "eip150Block": 0,
+      "eip155Block": 0,
+      "eip158Block": 0,
+      "byzantiumBlock": 0,
+      "constantinopleBlock": 0,
+      "petersburgBlock": 0,
+      "istanbulBlock": 0,
+      "berlinBlock": 0,
+      "londonBlock": 0,
+      "mergeNetsplitBlock": 0,
+      "depositContractAddress": "0x4242424242424242424242424242424242424242",
+      "terminalTotalDifficulty": 0,
+      "terminalTotalDifficultyPassed": true,
+      "shanghaiTime": 0,
+      "cancunTime": 0,
+      "blobSchedule": {
+        "prague": {
+          "target": 61,
+          "max": 91,
+          "baseFeeUpdateFraction": 13338477
```

Comment:
Could you also add a test where `baseFeeUpdateFraction` is not set (also not explicitly to `undefined`)? I think this will override and set the param in the fork to `undefined` instead of leaving that key alone.